File: online-judge-solutions/Codeforces/1716A.cpp
//https://codeforces.com/problemset/problem/1716/A
//Lang		:	GNU C++14
//Time		:	46 ms
//Memory	:	0 KB
#include<iostream>
#include<math.h>
using namespace std;
int main(){
	int t,n;
	cin>>t;
	while(t--){
		cin>>n;
		cout<<((n==1)?2:((n==2)||n==3?1:int(ceil(n/3.0))))<<"\n";
	}
	return 0;
}

File: online-judge-solutions/Codeforces/1716A.py
#https://codeforces.com/problemset/problem/1716/A
#Lang		:	Python 3.8
#Time		:	171 ms
#Memory         :	0 KB
import math
t=int(input())
for x in range(t):
    n=int(input())
    print(2 if n==1 else (1 if n==2 or n==3 else math.ceil(n/3)))
